src/openrct2/ride/coaster/meta/AirPoweredVerticalCoaster.h

```diff
@@ -25,6 +25,7 @@ constexpr const RideTypeDescriptor AirPoweredVerticalCoasterRTD =
     SET_FIELD(Flags, RIDE_TYPE_FLAGS_TRACK_HAS_3_COLOURS | RIDE_TYPE_FLAGS_COMMON_COASTER | RIDE_TYPE_FLAGS_COMMON_COASTER_NON_ALT | RIDE_TYPE_FLAG_PEEP_CHECK_GFORCES),
     SET_FIELD(RideModes, (1ULL << RIDE_MODE_POWERED_LAUNCH_PASSTROUGH) | (1ULL << RIDE_MODE_POWERED_LAUNCH)),
     SET_FIELD(DefaultMode, RIDE_MODE_POWERED_LAUNCH_PASSTROUGH),
+    SET_FIELD(OperatingSettings, { 30, 50,  30, 40,  40, 0 }),
```

Comment:
Many of these field assignment still have too many spaces after the comma. For example, this line should read:

```suggestion
    SET_FIELD(OperatingSettings, { 30, 50, 30, 40, 40, 0 }),
```